org.srb2.SRB2Kart.yaml

```diff
@@ -56,8 +68,8 @@ modules:
       - /lib/pkgconfig
     sources:
       - type: archive
-        url: https://github.com/libsdl-org/SDL_mixer/archive/refs/tags/release-2.0.4.tar.gz
-        sha256: 5605b6f230717acf6d09549671f7fe03f006700c61a61b86042888f81792e2b3
+        url: https://github.com/libsdl-org/SDL_mixer/archive/refs/tags/release-2.6.3.tar.gz
+        sha256: 91dd065e9e63f499e5317350b110184b0ba96bc5f63c39b3a9939a136c40c035
```

Comment:
Would be great to use the official module for SDL2 in [shared-modules](https://github.com/flathub/shared-modules/blob/master/SDL2/SDL2-with-libdecor.json) :pray:

Reply:
what is this exactly?

Reply:
There is a working example [here](https://github.com/flathub/org.openttd.OpenTTD). Instructions to work with `shared-modules` are [here](ttps://github.com/flathub/shared-modules).

Reply:
They are essentially a collection of community-maintained modules for popular libraries used in multiple manifests.

Reply:
no but why should this be added here? and isnt this sdl and not sdl-mixer?

Reply:
My bad, I misread it. Actually whilst looking for `SDL2_mixer` inside Flathub, I found out that it might already be included in the [Freedesktop SDK](https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/blob/master/elements/components/sdl2-mixer.bst).

Could you confirm whether that's the case, and check if the project builds fine with it ? We might as well get rid of the module if the runtime provides it.

Cheers

Reply:
maybe you forgot but the reason why we are rebuilding sdl mixer is to use modplug, see #38 .